### PR TITLE
Update AboutUs.adoc and UserGuide.adoc (Jian Wen)

### PR DESCRIPTION
**Pull request by lye-jw**
Updates:
- Add **`Lye Jian Wen`** section in `AboutUs.adoc`
- Add `update` and `clone` section in `UserGuide.adoc`
- Add `lye-jw.adoc`

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -17,4 +17,11 @@ image::limerencee.png[width="150", align="left"]
 Role: Project Manager/Developer +
 Responsibilities: Deliverables and Deadlines
 
+=== Lye Jian Wen
+image::lye-jw.png[width="150", align="left"]
+link:https://https://github.com/lye-jw[[github]] [<<lye-jw#, portfolio>>]
```
Comment (lightz96) [inline docs/AboutUs.adoc]: Your GitHub link is broken because there is 2 `https://` in your GitHub link

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -91,6 +91,50 @@ Expected output:
...
+* `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the Transaction is to be updated with.
+****
+
+Examples:
+
+*   `update i/1 | n/Curry Laksa`
+*   `update i/1 | c/11`
+
+Expected output: +
+** Assume 2nd command occurs after 1st*
```
Comment (lightz96) [inline docs/UserGuide.adoc]: There is an additional `*` in front. Is it intentional or accidental?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -91,6 +91,50 @@ Expected output:
...
 
+// tag::update[]
+=== Updating a Transaction’s details: `update`
+
+Format: `update i/INDEX | PREFIX_LETTER/UPDATED_DETAILS`
+
+****
+* Updates the Transaction's details at the specified `INDEX`.
+* `PREFIX_LETTER` specifies type of detail to be updated, e.g. `n/` for name.
+* `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the Transaction is to be updated with.
```
Comment (lightz96) [inline docs/UserGuide.adoc:105]: 1. Adds in what the `INDEX` referring to? index number that is currently shown in the UI?
2. States that `INDEX` must be a positive integer.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -91,6 +91,50 @@ Expected output:
...
+
+// tag::clone[]
+=== Cloning a Transaction: `clone`
+
+Format: `clone i/INDEX`
+
+****
+* Clones (creates a duplicate of) a Transaction.
+* Clone produced is added to a new index at the bottom of the list.
+****
```
Comment (lightz96) [inline docs/UserGuide.adoc:132]: 1. Adds in what the `INDEX` referring to? index number that is currently shown in the UI?
2. States that INDEX must be a positive integer.

If the entire user guide `INDEX` refers to the index number that is currently shown in the UI. We can add it somewhere at the top to indicate that it applied to the entire user guide.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -91,6 +91,50 @@ Expected output:
...
+
+Examples:
+
+*   `clone i/1`
+
+Expected output: +
+** Assume entry at index 1 is `1. [-] Laksa ($10.50) [Lunch][Food]` and there are no other entries.*
```
Comment (lightz96) [inline docs/UserGuide.adoc]: There is an additional `*` in front. Is it intentional or accidental?

**Review by lightz96 — CHANGES_REQUESTED** [review]
Looks great. Should be able to merge once the comments have been resolved. Please update your PR as there is a new merge.

File: docs/AboutUs.adoc (hunk 1 above)
Comment (lye-jw) [reply docs/AboutUs.adoc]: Noted, thanks!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (lye-jw) [reply docs/UserGuide.adoc]: Intentional to emphasize a point i.e. ***Take note of this**. Will remove and use `NOTE` instead.

Comment (lightz96) [reply docs/UserGuide.adoc]: You can consider using admonition from asciidoc. 

E.g. `NOTE: Assume...`

https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

File: docs/UserGuide.adoc (hunk 3 above)
Comment (lye-jw) [reply docs/UserGuide.adoc:105]: Noted!

File: docs/UserGuide.adoc (hunk 4 above)
Comment (lye-jw) [reply docs/UserGuide.adoc:132]: Good simplification, if this is the case after everyone has completed the UserGuide then we can do this modification.

**Comment by krusagiz** [comment]
I would suggest merging @kensonoenqh's documentation before merging this one.

**Review by lightz96 — APPROVED** [review]

**Pull request merged**